assets/src/components/supported-templates-toggle/index.js

```diff
@@ -9,20 +9,34 @@ import { __ } from '@wordpress/i18n';
  */
 import { Options } from '../options-context-provider';
 import { AMPSettingToggle } from '../amp-setting-toggle';
+import { AMPNotice } from '../amp-notice';
 
 export function SupportedTemplatesToggle() {
 	const { editedOptions, updateOptions } = useContext( Options );
 
-	const { all_templates_supported: allTemplatesSupported, theme_support: themeSupport } = editedOptions;
+	const { all_templates_supported: allTemplatesSupported, reader_theme: readerTheme, theme_support: themeSupport } = editedOptions;
 
-	return [ 'standard', 'transitional' ].includes( themeSupport ) ? (
-		<AMPSettingToggle
-			checked={ true === allTemplatesSupported }
-			text={ __( 'This will allow all of the URLs on your site to be served as AMP by default.', 'amp' ) }
-			title={ __( 'Serve all templates as AMP regardless of what is being queried.', 'amp' ) }
-			onChange={ () => {
-				updateOptions( { all_templates_supported: ! allTemplatesSupported } );
-			} }
-		/>
-	) : null;
+	const isReaderMode = 'reader' === themeSupport;
+	const isLegacy = isReaderMode && 'legacy' === readerTheme;
+
+	return (
+		<>
+			{ isLegacy && (
+				<AMPNotice>
+					<p>
+						{ __( 'This setting is not available when the legacy Reader theme is selected.', 'amp' ) }
```

Comment:
Content types shouldn't be able disabled as well if legacy mode is selected:

![image](https://user-images.githubusercontent.com/16200219/87100849-eee47f80-c23c-11ea-985b-4dafb6cbf22e.png)

Reply:
Ah nevermind, please disregard. This is OK :ok_hand:.

Reply:
How? The post types toggles _should_ be able to be available in legacy Reader mode. It is the only such customization option available. Only the template hierarchy should be hidden (and it is).